provider/cluster/kube/client.go

```diff
@@ -136,6 +139,93 @@ func openKubeConfig(cfgPath string, log log.Logger) (*rest.Config, error) {
 	return rest.InClusterConfig()
 }
 
+func (c *client) ClearHostname(ctx context.Context, ownerAddress cosmostypes.Address, dseq uint64, hostname string) error {
+	labelSelector := &strings.Builder{}
+	_, _ = fmt.Fprintf(labelSelector, "%s=%s", akashLeaseOwnerLabelName, ownerAddress.String())
+	_, _ = fmt.Fprintf(labelSelector, ",%s=%d", akashLeaseDSeqLabelName, dseq)
```

Comment:
Construct one from a map then serialize it to a string.  See [here](https://github.com/ovrclk/akash/blob/9f13194fa52ab1c2d0abec149aba284aa87735d1/provider/cluster/kube/builder.go#L536-L540) for constructing one.